src/AzureCloudKeyVault.Tests/AzureKeyVaultResourceTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Squadron.AzureKeyVault.Tests
+{
+    public class AzureKeyVaultCreateNewResourceTests
+        : IClassFixture<AzureKeyVaultResource<AzureKeyVaultNewOptions>>
+    {
+        private readonly AzureKeyVaultResource<AzureKeyVaultNewOptions> _resource;
+
+        public AzureKeyVaultCreateNewResourceTests(
+            AzureKeyVaultResource<AzureKeyVaultNewOptions> resource)
+        {
+            _resource = resource;
+        }
+
+        [Fact(Skip = "Can not run without Azure credentials")]
+        public async Task PrepareVault_CreateNew_NoError()
```

Comment:
If the only test is skipped... maybe fake some azure credentials?  adding an appsettings.json in the test project and that should override the default one... otherwise maybe add another test?

Reply:
Hard to test the test framework here.
Wo yould add azure credentials on the CI build in the future to have a running test for this.